### PR TITLE
Fix #505: The "permalink" button in probe-dictionary should be bold

### LINKED ISSUE #505: The "permalink" button in probe-dictionary should be bold
In the top menu bar, all other labels are bold. The "Get permalink" button should be bold too.

### PR DESCRIPTION
**Pull request by Insiyaa**
To match with the style of other buttons, permalink button should be bold too.

### CODE REVIEW

**Comment by Insiyaa** [comment]
Also I'd propose the Probe Dictionary to be bold as well, should it go with this PR?... Please review!
tt

**Comment by georgf** [comment]
Good point, that seems to look more consistent. Thanks!

File: probe-dictionary/explorer.css [hunk 1]
```diff
@@ -14,6 +14,7 @@ body {
     padding: 7.5px 8px;
     line-height: 1.5;
     color:rgba(255,255,255,.75);
+    font-weight: 700;
```
Comment (georgf) [inline probe-dictionary/explorer.css]: We just use `font-weight: bold;` for `.nav-item`, let's use the same for this button.

**Review by georgf — COMMENTED** [review]
Thanks! Can you provide a [live link](https://github.com/mozilla/telemetry-dashboard/#submitting-pull-requests-for-the-dashboard) to how this looks so it's easier to check?

**Comment by Insiyaa** [comment]
I'm making changes to a branch, so I can't share a demo with gh-pages... Please guide me with some other way or should I share the screenshot?

**Comment by Sylvia23** [comment]
Hi @Insiyaa , I believe that the changes should be visible in the `live link` (ie the old link that you were using) of `your repository` as you have pushed the `branch` to ` origin`. You can check that out by opening the link itself. 
Thanks!

**Comment by Insiyaa** [comment]
@Sylvia23 Thanks! But changes were not there as those changes weren't made to gh-pages branch. But since I closed the prev PR, I merged the branch to gh-pages and so the changes are updated...

**Comment by Insiyaa** [comment]
Link: https://insiyaa.github.io/telemetry-dashboard/probe-dictionary/

**Review by georgf — APPROVED** [review]
Thanks, that looks good!

**Pull request merged**